### PR TITLE
Fuzz isa_a with spike-ffi

### PR DESCRIPTION
**Pull request by mohanson**
- Implement spike-ffi
- Fuzz A extension between interpreter, asm and spike.

```sh
$ cd fuzz_spike_ffi
$ sh build.sh
$ cd ..

$ cargo +nightly fuzz run isa_a
```

### CODE REVIEW

File: fuzz_spike_ffi/build.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+set -ex
```
Comment (XuJiandong) [inline fuzz_spike_ffi/build.sh]: Add `readme` about how to build fuzz_spike_ffi crate and how it works.

**Review by XuJiandong — CHANGES_REQUESTED** [review]

**Comment by xxuejie** [comment]
IMHO it might be better to put `fuzz_spike_ffi` in `ckb-vm-test-suite`

**Comment by mohanson** [comment]
I created a crate so that ckb-vm only needs to import it into dependency

source: https://github.com/nervosnetwork/ckb-vm-test-suite/tree/master/spike-sys
crates.io https://crates.io/crates/spike-sys

cc @XuJiandong @xxuejie

**Review by XuJiandong — DISMISSED** [review]

File: fuzz/Cargo.toml [hunk 2]
```diff
@@ -9,16 +9,19 @@ edition = "2018"
...
+[dependencies.ckb-vm]
+path = ".."
+features = ["asm"]
 
 # Prevent this from interfering with workspaces
 [workspace]
 members = ["."]
 
 [[bin]]
-name = "asm"
```
Comment (xxuejie) [inline fuzz/Cargo.toml]: Why do we replace one with the other, instead of keeping both fuzzing targets? I thought they can work for different use cases.

Comment (mohanson) [reply fuzz/Cargo.toml]: I added it back to the fuzz targets. It tests the load_program function which I overlooked earlier.

**Review by xxuejie — APPROVED** [review]

**Pull request merged**